lightningd/lightningd.h

```diff
@@ -289,4 +292,11 @@ void test_subdaemons(const struct lightningd *ld);
 /* Notify lightningd about new blocks. */
 void notify_new_block(struct lightningd *ld, u32 block_height);
 
+/* Signal a clean exit from lightningd.
+ * NOTE! This function **returns**.
+ * This just causes the main loop to exit, so you have to return
+ * all the way to the main loop.
```

Comment:
```suggestion
 * all the way to the main loop for `lightningd` to exit
```

This is a bit tautological, as written.